examples/zh/README-CN.md

```diff
@@ -0,0 +1,45 @@
+[English](../README.md) | 简体中文
+
+# 示例
+
+这些示例默认 `kustomize` 在您的 `$PATH` 中。
+
+这些示例通过了 [pre-commit](../bin/pre-commit.sh) 测试，并且应该与 HEAD 一起使用。
+
+<!-- @installkustomize @test -->
+```
+go get sigs.k8s.io/kustomize
+```
+
+ * [hello world](helloWorld/README.md) - 部署多个不同配置的 Hello World 服务。
```

Comment:
helloWorld/README.md --> ../helloWorld/README.md

Same for the following links.

Reply:
My bad